libs/@local/graph/postgres-store/src/store/postgres/query/condition.rs

```diff
@@ -65,23 +66,24 @@ impl Transpile for Condition {
                 Ok(())
             }
             Self::Not(condition) => {
-                fmt.write_str("NOT(")?;
-                condition.transpile(fmt)?;
-                fmt.write_char(')')
+                if let Self::Exists(path) = &**condition {
```

Comment:
This looks like it doesn't handle nested paths, is that correct? As in: `not(all([exists(...), ...]))` would fail.

Reply:
Great just wanted to verify. NULL in SQL is a bit weird so I wasn't sure.

Reply:
It is indeed...